xtask/Cargo.toml

```diff
@@ -2,9 +2,15 @@
 name = "xtask"
 version = "0.1.0"
 edition = "2021"
-rust-version = "1.83"
+rust-version = "1.85"
 publish = false
 
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = [
+    # Added by xshell
+    'cfg(trick_rust_analyzer_into_highlighting_interpolated_bits)',
```

Comment:
question: wat is this

I see <https://github.com/matklad/xshell/blob/52f71bac326aaac291d07146ea790ad886dd8131/src/lib.rs#L342> with the same wording, but that doesn't seem to actually be a `cfg`.

Reply:
https://github.com/matklad/xshell/blob/v0.2.6/src/lib.rs Seems only be a thing in 0.2.6...